src/styles/components/_Discovery.scss

```diff
@@ -1,21 +1,22 @@
 .discoveryContainer {
   margin: 1em auto;
   width: 100%;
+  text-align: center;
 
   .discoveryBox {
     position: relative;
+    text-align: -webkit-center !important;
```

Comment:
can we do this w/o the !important?

Reply:
also would reg. center work? webkit is just for chrome support

Reply:
yes, i will update.